features/support/tag_finder.rb

```diff
@@ -0,0 +1,21 @@
+# Finds Git tags, returns them as TagList instances
+class TagFinder
+
+  # Returns all tags (local + remote)
+  def self.all_tags
```

Comment:
This isn't just returning the tags but is returning the tags as a table. The comment and method name should reflect that